release-notes.md

```diff
@@ -24,33 +24,40 @@ We also provide nightly **ossedge** builds from OSS master without Armory featur
 ```yml
 $ hal version list
 ...
-- 2.0.1 (OSS 1.9.x):
-  Published: Fri October 12 15:42:49 PDT 2018
-  Changelog: https://docs.armory.io/release-notes
-- 2.0.2 (OSS 1.9.x):
-  Published: Fri October 12 15:42:49 PDT 2018
-  Changelog: https://docs.armory.io/release-notes
+ - 2.0.0 (OSS Release 1.9.5):
+   Changelog: https://docs.armory.io/release-notes/armoryspinnaker_v2.0.0/
+   Published: Fri Nov 02 16:42:47 PDT 2018
+   (Requires Halyard >= 1.2.0)
+ - 2.0.1 (OSS Release 1.9.5):
+   Changelog: https://docs.armory.io/release-notes/armoryspinnaker_v2.0.1/
+   Published: Fri Dec 14 18:32:56 PST 2018
```

Comment:
the published dates for v2.0.1 seems to have changed

Reply:
What do you mean? I got it from `hal version list`. The previous date may have been made up.

Reply:
2.0.2 doesn't exist. It was made up to not look empty

Reply:
ah gotcha!